src/routes/forumComments/editForumComment.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+const mysql = require('mysql');
+const config = require('../../../config.js');
+
+//this edits an exising forum's information given the forum ID
+router.post('/api/editForumComment', (req, res) => {
```

Comment:
change this to a PUT method, refer to previous discussions for implementation

Reply:
working on postman after changing to `PUT`:
![image](https://user-images.githubusercontent.com/57375371/225399669-3ac7498d-d0c8-490b-96e8-278035089b2f.png)